### DISCUSSION TITLE
关于`rc.2`开始，打包工具默认需要用户手动安装的困惑

### DISCUSSION THREAD

**Discussion by pengzhanbo** [vuepress/core#1478] [General]
在最近的 `rc.2` 的更新中，打包工具 由原来的 默认使用 `vite bundler`，变更为了 需要用户手动安装 `vite bundler` 或 `webpack bundler`，是出于什么考量的？

按我的猜测，是否是出于 减少用户安装 `vuepress` 软件包所需要的时间，以及占用的磁盘空间 的角度来考虑的？
但是按我的理解，是否应该是优先考虑减少用户的使用成本？ 

虽然默认不安装某个 `bundler` 可以减少一点安装时间和磁盘空间占用，但是对用户来说，却增加了配置的步骤，这带来了使用成本的增加。

**Comment by Mister-Hope** [comment]
这种 trade off 我认为是可以理解的，用户只需要一次配置，而安装可能在ci中多次进行。
